meteor-app/imports/ui/layouts/app.js

```diff
@@ -21,7 +21,7 @@ import JobHistory from '/imports/ui/assessment/assessment-job-history'
 import MemberAddContainer from '/imports/ui/member/member-add-container'
 import MemberMainContainer from '/imports/ui/member-main-container'
 import MemberEdit from '/imports/ui/member-edit'
-import Visit from '/imports/ui/visit'
+import Visit from '/it'
```

Comment:
This looks like a mistake - importing a file from /it ?

Reply:
Doh!